tools/msvs/install_tools/install_tools.bat

```diff
@@ -36,7 +36,6 @@ echo result of a Chocolatey install. This acceptance occurs whether you know the
 echo license terms or not. Read and understand the license terms of the packages
 echo being installed and their dependencies prior to installation:
 echo - https://chocolatey.org/packages/chocolatey
-echo - https://chocolatey.org/packages/boxstarter
 echo - https://chocolatey.org/packages/python2
 echo - https://chocolatey.org/packages/visualstudio2017buildtools
```

Comment:
Based on the Chocolatey packages being installed, this one is no longer being directly installed, so I am assuming that it is coming in as a dependency.  Since you are not listing all the dependent packages, I don't think it is right to explicitly call this one out.